apps/graphql/src/dataloaders/bookingsLoader/BookingsLoader.js

```diff
@@ -109,15 +110,18 @@ const sanitizeMulticity = (booking: BookingApiResult) => {
 
   return {
     segments,
-    trips: Object.keys(trips).map(key => trips[key]),
+    trips: Object.keys(trips).reduce((acc, key) => {
+      return [...acc, ...trips[key]];
+    }, []),
     departure: head(segments)?.departure,
     arrival: last(segments)?.arrival,
   };
 };
 
-const sanitizeFlight = (flight: ApiFlight) => {
+const sanitizeFlight = (flight: ApiFlight): Segment => {
```

Comment:
Probably not issue for this PR, but this Segment could be also more consistent with Segment from search results.

Reply:
Yes, I noticed, I want to change Segment from SearchResult to this structure, but in a later PR